engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-traces-modal.controller.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.content-hosts.controller:ContentHostsBulkTracesController
+ *
+ * @requires $scope
+ * @requires $state
+ * @requires $timeout
+ * @requires $window
+ * @requires $uibModalInstance
+ * @requires HostBulkAction
+ * @requires Organization
+ * @requires CurrentOrganization
+ * @requires Notification
+ * @requires Nutupane
+ * @requires BastionConfig
+ * @requires HostTraces
+ * @requires hostIds
+ *
+ * @description
+ *   Provides the functionality for the content host package list and actions.
+ */
+/*jshint camelcase:false*/
+angular.module('Bastion.content-hosts').controller('ContentHostsBulkTracesController',
+    ['$scope', '$state', '$timeout', '$window', '$uibModalInstance', 'HostBulkAction', 'Organization', 'CurrentOrganization', 'Notification', 'Nutupane', 'BastionConfig', 'HostTraces', 'hostIds',
+    function ($scope, $state, $timeout, $window, $uibModalInstance, HostBulkAction, Organization, CurrentOrganization, Notification, Nutupane, BastionConfig, HostTraces, hostIds) {
+
+        var tracesNutupane = new Nutupane(HostBulkAction, hostIds, 'traces');
+        tracesNutupane.enableSelectAllResults();
+        tracesNutupane.masterOnly = true;
+        $scope.controller = 'host_traces';
```

Comment:
I have a feeling this doesn't do anything - see if things still work without it

Reply:
Removed it.